dev/tools/examples_smoke_test.dart

```diff
@@ -17,21 +17,21 @@ import 'package:path/path.dart' as path;
 import 'package:platform/platform.dart';
 import 'package:process/process.dart';
 
-FileSystem filesystem = const LocalFileSystem();
-ProcessManager processManager = const LocalProcessManager();
-Platform platform = const LocalPlatform();
+const FileSystem _kFilesystem = LocalFileSystem();
```

Comment:
This change is just because VSCode keeps trying to import this file in tests when I type "filesystem", and it shouldn't be public anyhow.